src/StoreKeeper/WooCommerce/B2C/Exports/OrderExport.php

```diff
@@ -20,6 +20,9 @@
 
 class OrderExport extends AbstractExport
 {
+    const EMBALLAGE_TAX_RATE_ID_META_KEY = 'storekeeper_emballage_tax_id';
+    const IS_EMBALLAGE_FEE_KEY = 'is_emballage_fee';
+    const TAX_RATE_ID_FEE_KEY = 'sk_tax_rate_id';
```

Comment:
```suggestion
    const EMBALLAGE_TAX_RATE_ID_META_KEY = 'sk_emballage_tax_id';
    const IS_EMBALLAGE_FEE_KEY = 'sk_is_emballage_fee';
    const TAX_RATE_ID_FEE_KEY = 'sk_tax_rate_id';
```

Let's have same prefix for all

Reply:
Changes implemented.